multiapps-controller-web/src/main/java/org/cloudfoundry/multiapps/controller/web/security/SpaceGuidBasedAuthorizationFilter.java

```diff
@@ -3,36 +3,53 @@
 import java.io.IOException;
 import java.text.MessageFormat;
 
+import javax.inject.Inject;
+import javax.inject.Named;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+import org.cloudfoundry.multiapps.controller.core.auditlogging.LoginAttemptAuditLog;
 import org.cloudfoundry.multiapps.controller.web.Messages;
 import org.cloudfoundry.multiapps.controller.web.util.SecurityContextUtil;
 import org.cloudfoundry.multiapps.controller.web.util.ServletUtil;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.web.server.ResponseStatusException;
 
+@Named
 public abstract class SpaceGuidBasedAuthorizationFilter implements UriAuthorizationFilter {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(SpaceGuidBasedAuthorizationFilter.class);
 
     private final AuthorizationChecker authorizationChecker;
 
+    @Inject
+    private LoginAttemptAuditLog loginAttemptAuditLog;
+
     protected SpaceGuidBasedAuthorizationFilter(AuthorizationChecker authorizationChecker) {
         this.authorizationChecker = authorizationChecker;
     }
 
+    //For unit tests
+    protected SpaceGuidBasedAuthorizationFilter(AuthorizationChecker authorizationChecker, LoginAttemptAuditLog loginAttemptAuditLog) {
+        this.authorizationChecker = authorizationChecker;
+        this.loginAttemptAuditLog = loginAttemptAuditLog;
+    }
+
     @Override
     public final boolean ensureUserIsAuthorized(HttpServletRequest request, HttpServletResponse response) throws IOException {
         String spaceGuid = extractAndLogSpaceGuid(request);
-        auditLogLoginAttempt(SecurityContextUtil.getUsername(), spaceGuid, Messages.USER_TRYING_TO_LOGIN_AUDIT_LOG_MESSAGE);
```

Comment:
I don't see removal of `auditLogLoginAttempt` method from interface and deletion of `AuditLoggingProvider` class.